routes/api/v1/widget_specifications.js

```diff
@@ -0,0 +1,37 @@
+const express = require( 'express' )
+const router = express.Router()
+const errorResponse = require( '../src/errorResponse')
+
+const WidgetSpecification = require( '../../../models/widget_specifications' )
+
+router.get( '/', ( request, response ) => {
+  WidgetSpecification.find().exec()
+    .then( widget_specifications => response.json( widget_specifications ))
+    .catch( errorResponse( response ))
+})
+
+const insertWidgetSpecification = data => {
```

Comment:
Let's move this into another module and import - we should probably create a folder in this directory specifically for utility functions like this.